Dockerfile

```diff
@@ -174,6 +181,9 @@ WORKDIR /elpis-gui
 RUN npm install && \
     npm run build
 
+# For /elpis (elan conversion)
+RUN npm install xslt3
+
```

Comment:
This should be included in the package.json and installed with the above command, so this RUN command can be removed.